src/components/Modals/SaveAsModal.vue

```diff
@@ -22,7 +26,8 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
                 :value="saveWhat.toLowerCase()"
                 spellcheck="false"
               />
-              <button class="download-button" @click="downloadFile">download</button>
+              .json
+              <button class="download-button" @click="downloadFile">DOWNLOAD</button>
```

Comment:
I read somewhere that it's best practice to use lowercase in your markup and then use CSS to make the text uppercase, but I am not sure. Totally your call

Reply:
Okay, I trust you